weaviate/connect/v4.py

```diff
@@ -102,7 +105,22 @@ def __init__(
         connection_config: ConnectionConfig,
         loop: asyncio.AbstractEventLoop,  # required for background token refresh
         embedded_db: Optional[EmbeddedV4] = None,
+        additional_config: Optional[AdditionalConfig] = None,
```

Comment:
The addition of this new parameter serves not purpose in this PR. The new attribute `self._additional_config` is unused so I think that this change is needlessly confusing